build/docker/zap-full-scan.py

```diff
@@ -420,6 +424,10 @@ def main(argv):
                 # Save the report
                 write_report(base_dir + report_html, zap.core.htmlreport())
 
+            if report_json:
+                # Save the report
+                write_report(base_dir + report_json, zap.core.jsonreport())
```

Comment:
Ah! Its a post 2.6.0 change isnt it :/
It will still work on the weekly (which is what this script uses), so we could use the underlying method to call the endpoint.